examples/cms-enterspeed/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "classnames": "2.3.1",
+    "date-fns": "2.28.0",
+    "next": "latest",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "autoprefixer": "10.4.2",
+    "postcss": "8.4.5",
+    "tailwindcss": "^3.0.15"
```

Comment:
You will need to add a few dependencies `typescript`, `@types/react` and `@types/node` here.

Also, a `tsconfig.json` file should be generated on `next dev` which should also be committed. :+1: